src/util.ts

```diff
@@ -14,15 +14,17 @@ limitations under the License.
 import * as http from "http";
 import * as https from "https";
 import * as fileType from "file-type";
-import { Buffer } from "buffer";
+import {Buffer} from "buffer";
```

Comment:
what did you do to all the extra spaces on imports D: they were there on purpose!